environments/jvm/pom.xml

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+
+	<groupId>io.fission</groupId>
+	<artifactId>jvmenv</artifactId>
```

Comment:
To keep the naming consistent, use `io.fission.env-jvm` or `io.fission.env-spring` or `io.fission.env-java` (see comment about environment naming